internal/llrp/device_service_test.go

```diff
@@ -342,10 +237,7 @@ func TestSetConfig(t *testing.T) {
 				assert.NotNil(tt, errMsg, "Encountered Error: %s", errMsg)
 			}
 
-			s.Close()
```

Comment:
I see code like this a few times in this file:
```go
			errMsg := deviceServiceClient.SetConfig(tc.deviceName, se)
			if tc.respCode == http.StatusOK {
				assert.Nil(tt, errMsg, "Encountered Error: %s", errMsg)
			} else {
				assert.NotNil(tt, errMsg, "Encountered Error: %s", errMsg)
```

I'm not sure if the return type changed, but I think these blocks of code should be updated to something like:
- `errMsg` should be `err`
- Use `if errorExpected` instead of checking `tc.respCode`
- Use `assert.Error` and `assert.NoError`
```go
			err := deviceServiceClient.SetConfig(tc.deviceName, se)
			if errorExpected {
				assert.Error(tt, err)
			} else {
				assert.NoError(tt, err)
```

Reply:
YES!! Good catch!

Reply:
I made the suggested change